Misc/NEWS.d/next/Tests/2018-05-28-10-58-39.bpo-32374.NErc2R.rst

```diff
@@ -0,0 +1 @@
+Test added with pull #5140 now does not pass on Python exception raise.
```

Comment:
I don't think changes to the testsuite need a NEWS entry – and if they did, pull #5140 isn't acceptable shorthand.

Reply:
It is my understanding that test suite fixes generally do get news entries, since they can affect whether or not python's test suite passes when someone is testing a deployment or vendor package.  However, they aren't needed unless the test was introduced in one release and fixed in another (but that does include crossing alpha/beta/release candidate boundaries).